src/Compilers/CSharp/Portable/Parser/Lexer_StringLiteral.cs

```diff
@@ -443,6 +443,23 @@ private void ScanInterpolatedStringLiteralContents(ArrayBuilder<Interpolation>?
 
             private void ScanFormatSpecifier()
             {
+                /*
+                ## Grammar from spec:
+                
+                interpolation_format
+                    : ':' interpolation_format_character+
+                ; 
+                interpolation_format_character
+                    : '<Any character except \" (U+0022), : (U+003A), { (U+007B) and } (U+007D)>'
+                ;
+
+                ------------------
+                Even though the spec says that ", :, { and } are prohibited in the format specifier
```

Comment:
Unlike Cyrus, I think this addendum is actively harmful. It reads like something we're doing currently. I think just the spec part is fine and won't lead to future confusion.